File: sem4/MPP/proiect/appClientServer/CompetitionNetworking/src/main/java/org/example/dto/DTOUtils.java
package org.example.dto;

import org.example.model.Task;

public class DTOUtils {
    public static TaskDTO getDTO(Task event) {
        return null;
    }
    public static TaskDTO[] getDTO(Task[] tasks) {
        TaskDTO[] frDTO = new TaskDTO[tasks.length];
        for(int i=0; i<tasks.length; i++)
            frDTO[i] = getDTO(tasks[i]);
        return frDTO;
    }
}

File: sem4/MPP/proiectC#/AppClientServer/Networking/DTOUtils.cs
using model;

namespace Networking;

public class DTOUtils
{
    public static Task getFromDTO(TaskDTO usdto)
    {
        string desc =usdto.description;
        int ageMin = usdto.ageCatStart;
        int ageMax = usdto.ageCatEnd;
        return new Task(desc,ageMin,ageMax);

    }

}